dockerfiles/che/Dockerfile

```diff
@@ -9,7 +9,7 @@
 #   Codenvy S.A
 #
 # To build it, run in the repository root:
-#  `docker build -t codenvy/che-server .`
+#  `docker build -t eclipse/che-server .`
```

Comment:
This doesn't work anymore. I would rather:
```
docker build -t eclipse/che-server -f dockerfiles/che/Dockerfile .
```
from the root of the repo. 
Dockerfile could copy assembly from `assembly/assembly-main/target/eclipse-che-*/eclipse-che-*` as before.